build-tools/packages/build-tools/src/repoPolicyCheck/handlers/index.ts

```diff
@@ -9,11 +9,13 @@ import { handler as dockerfilePackageHandler } from "./dockerfilePackages";
 import { handler as fluidCaseHandler } from "./fluidCase";
 import { handlers as lockfilesHandlers } from "./lockfiles";
 import { handlers as npmPackageContentsHandlers } from "./npmPackages";
+import { handler as peerDepsHandler } from "./peerDependencies";
 
 /**
  * declared file handlers
  */
 export const policyHandlers: Handler[] = [
+    peerDepsHandler,
```

Comment:
does this really need to be in the global policy check? for things as simple as this i usually just add a unit test in the package the reads the package.json and asserts the condition. this keeps the policy co-located with the package and makes fixing, changing, and understanding it easier

Reply:
That's a good point; I agree that a global policy handler seems extreme here. A unit-test is a good idea. I'll explore that.